servers/login/src/main/kotlin/fr/rob/login/security/authentication/AuthenticationOpcode.kt

```diff
@@ -1,15 +1,34 @@
 package fr.rob.login.security.authentication
 
+import fr.rob.core.network.Packet
 import fr.rob.core.network.session.Session
 import fr.rob.core.opcode.ProtobufOpcodeFunction
 import fr.rob.core.process.ProcessManager
+import fr.rob.entities.AuthenticationProto
+import fr.rob.login.opcode.ServerOpcodeLogin
 
 abstract class AuthenticationOpcode(private val processManager: ProcessManager) :
     ProtobufOpcodeFunction(false) {
 
     override fun call(session: Session, message: Any) {
         val authenticationProcess = processManager.makeProcess(AuthenticationProcess::class)
 
-        authenticationProcess.authenticate(session, message)
+        val authenticationResult = AuthenticationProto.AuthenticationResult.newBuilder()
+
+        if (authenticationProcess.authenticate(session, message)) {
+            authenticationResult.result = AUTHENTICATION_RESULT_SUCCESS
+        } else {
+            authenticationResult.result = AUTHENTICATION_RESULT_ERROR
+            authenticationResult.code = AUTHENTICATION_CODE_BAD_CREDENTIALS
```

Comment:
I think it's cleaner to handle this in the process and retrieve the message by `authenticationProcess.getError()` or `authenticationProcess.error`.

The error constants are stored in the process and can be reuse in multiple places unlike in the opcode function